t/cli/test_admin.sh

```diff
@@ -112,6 +113,67 @@ if [ $count -eq 0 ]; then
     exit 1
 fi
 
+echo "
+apisix:
+    enable_admin: true
+    allow_admin:
+        - 127.0.0.0/8
+" > conf/config.yaml
+
+make init
+
+count=`grep -c "allow 127.0.0.0/8" conf/nginx.conf`
+if [ $count -eq 0 ]; then
+    echo "failed: not found 'allow 127.0.0.0/8;' in conf/nginx.conf"
+    exit 1
+fi
+
+echo "
+apisix:
+    enable_admin: true
+    allow_admin:
+        - ::1
```

Comment:
Could you rewrite or remove test cases which are not relative to the new feature?

Reply:
To make some confirmation, the not relative is that `allow_admin` should be limited on which option?

1. Option A `IPv4 address` and `IPv4 CIDR`, the others are not relative.
2. Option B `IPv6 address` and `IPv6 CIDR` are not relative, the others are ok.
3. Option C `all` is not relative.

[allow](https://nginx.org/en/docs/http/ngx_http_access_module.html#allow) (ngx_http_access_module)

```
Syntax: | allow address | CIDR | unix: | all;
```